tests/jerry/es2015/typedarray-prototype-tolocalestring.js

```diff
@@ -0,0 +1,77 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var int_typedarrays = [
```

Comment:
There is no test where the element's `toLocaleString` is invoked as required by the standard. For that the number prototype's toLocaleString should be overwritten.